spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/test/domain/TradeRepository.java

```diff
@@ -35,6 +35,23 @@
 
 	int countByAction(String action);
 
+	@Query(" select count(1) from :org.springframework.cloud.gcp.data.spanner.test.domain.Trade: "
+			+ "where action = @action")
+	int countByActionQuery(@Param("action") String action);
+
+	@Query("SELECT EXISTS(select * from "
+			+ ":org.springframework.cloud.gcp.data.spanner.test.domain.Trade: "
+			+ "where action = @action limit 1)")
+	boolean existsByActionQuery(@Param("action") String action);
+
+	@Query("select action from :org.springframework.cloud.gcp.data.spanner.test.domain.Trade: "
+			+ "where action = @action limit 1")
+	String getFirstString(@Param("action") String action);
+
+	@Query("select action from :org.springframework.cloud.gcp.data.spanner.test.domain.Trade: "
+			+ "where action = @action")
+	List<String> getFirstStringList(@Param("action") String action);
```

Comment:
Can we also add one that returns a single `Trade`?

Reply:
@meltsufin PTAL. I added an integration test method for just a single POJO and also added these to the sample.